zebrad/src/components/mempool/error.rs

```diff
@@ -0,0 +1,29 @@
+//! Errors that can occur when adding transactions to the mempool or querying it.
+
+use thiserror::Error;
+
+#[derive(Error, Debug, PartialEq)]
```

Comment:
Optional: We might want to be able to copy or clone these errors.
```suggestion
#[derive(Error, Copy, Clone, Debug, PartialEq)]
```

Reply:
Oops, sorry, this didn't work, because the error types we're wrapping don't implement `Copy` or `Clone`.

I'll see if I can fix that now.

Reply:
I made a comment with a suggested fix.